src/pages/Categories.js

```diff
@@ -1,9 +1,24 @@
 import React from 'react';
+import { useDispatch } from 'react-redux/es/exports';
+import { checkStatusActionCreator } from '../redux/categories/categories';
 
-const Categories = () => (
-  <div className="container">
-    <button type="button"> Check Status </button>
-  </div>
-);
+const Categories = () => {
+  const dispatch = useDispatch();
+  const onClickHandle = () => {
+    dispatch({
+      type: checkStatusActionCreator().type,
+    });
+  };
+  return (
+    <div className="container">
+      <button
+        type="button"
+        onClick={onClickHandle}
+      >
+        Check Status
+      </button>
+    </div>
+  );
+};
```

Comment:
- [ ] After dispatching the action to the categories reducer and modifying the category state you need to fetch the category state using `useSelector` and display the text on this component.